CLI_TEST_CHECKLIST.md

```
@@ -0,0 +1,302 @@
+# CLI Testing Checklist
+
+# CLI Testing Checklist
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)